command/operator_debug.go

```diff
@@ -54,14 +55,17 @@ type OperatorDebugCommand struct {
 	cancel        context.CancelFunc
 	opts          *api.QueryOptions
 	verbose       bool
+	members       *api.ServerMembers
+	nodes         []*api.NodeListStub
 }
 
 const (
-	userAgent   = "nomad operator debug"
-	clusterDir  = "cluster"
-	clientDir   = "client"
-	serverDir   = "server"
-	intervalDir = "interval"
+	userAgent                     = "nomad operator debug"
+	clusterDir                    = "cluster"
+	clientDir                     = "client"
+	serverDir                     = "server"
+	intervalDir                   = "interval"
+	minimumVersionPprofConstraint = ">= 0.11.0, <= 0.11.2"
```

Comment:
could rename to `restrictedVersionPprofConstraint` but we'll probably want to restrict on all versions before 0.11 soon anyways